### DISCUSSION TITLE
Install with Docker  - Database "acore_auth" does not exist

### DISCUSSION THREAD

**Discussion by guilhermeandrea** [azerothcore/azerothcore-wotlk#13074] [General]
when i run the containers, this problem appears : 

![docker problem](https://user-images.githubusercontent.com/110643891/191324193-be8e823d-8228-427b-bfdb-e37323cdcfe1.png)

What can i do to fix?

**Comment by MrLavender** [comment]
Just attach to the worldserver container and type `yes` to create it.

The `./acore.sh docker attach ac-worldserver` command in the instructions doesn't work for me so I used `docker attach azerothcore-wotlk-ac-worldserver-1`.
